src/context.cpp

```diff
@@ -943,6 +952,54 @@ namespace librealsense
             throw std::runtime_error( "Could not find a stream that supports option " + name );
         }
 
+        void add_processing_block( std::string filter_name )
```

Comment:
This file starts to be really long. If I read it correctly, most of it is dds_sensor_proxy class - please consider to add new h and cpp files for it.

Reply:
Already mentioned in a previous PR (about metadata support) that we intend to split the DDS to a different file, but because the changes we made broke the LibCI we address it first. After changes will be merged we will create a new PR to split this file.

Reply:
ok